### DISCUSSION TITLE
Minibar: ["all_at_once"] = true

### DISCUSSION THREAD

**Discussion by WS64** [koreader/koreader#7086] [Q&A] [answered]
How do I enable in the GUI in settings.reader.lua this setting:
    ["footer"] = {
        ["all_at_once"] = true,
I can't find that one in the GUI

**Comment by NiLuJe** [comment, marked as answer]
There's a dedicated "Status bar" menu in  the "Gear" tab while in CRe. "Show all at once" is under "Settings" in there.

**Reply by WS64** [reply]
Thx, I did not see the second page!
